### PR TITLE
Styling: Fix some minor spelling error and formatting.

### PR DESCRIPTION
**Pull request by jannylund**
Fix a wrongly spelled variable name + some other minor issues.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ class TelldusTDToolPlatform {
   }
 }
 
-/*
+/**
```
Comment (amlinger) [inline src/index.js:71]: Not too sure what difference this would make, but no biggie.

**Review by amlinger — APPROVED** [review]
Thanks for taking the time to correct this! Always good to have properly named variables and properly formatted code.

Awesome stuff! :)

File: src/index.js (hunk 1 above)
Comment (jannylund) [reply src/index.js:71]: The difference is how it's rendered in editor/IDE, as well as to be consistent with any other comments in the source.

screenshot 2016-10-09 22 54 58

Comment (amlinger) [reply src/index.js:71]: The difference in that case would be that other comments using double-star in the beginning is meant for documenting functions, whereas this only documents what happens in the following lines, not the interface that could be interesting for consumers.

However, I don't mind keeping this anyway :)

**Pull request merged**